app/controllers/organization_campaigns_controller.rb

```diff
@@ -1,13 +1,24 @@
 class OrganizationCampaignsController < ApplicationController
-  before_action :store_campaign
   before_action :authenticate_user!
 
   def show
+    store_campaign
     @recipients = @organization_campaign.recipients
     @donors = @organization_campaign.organization.memberships
     @match = Match.new
   end
 
+  def create
+    # Create a linkage between org and campaign.  If the linkage already 
+    # exists, ignore it.  Otherwise send an email to the Volunteer Center
+
+    OrganizationCampaign.find_or_create_by(organization_campaign_params) do |oc|
+      # TODO: this is called if created.  Send email
+    end 
+    redirect_to controller: :organizations, action: :show, 
+      id: params[:organization_campaign][:organization_id]
+  end
+
```

Comment:
I think we can DRY this up a little with route helpers. pseudo-code:
```ruby
    @org_camp =  OrganizationCampaign.find_or_create_by(organization_campaign_params) do |oc|
      # TODO: this is called if created.  Send email
     end
    redirect_to @org_camp.organization
```
Currently unable to test this due to bug on organization show